# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../samba/fix_smbadduser_path.patch
# Copyright (C) 2006 - 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./source3/smbadduser.in.old	2006-11-01 15:38:14.000000000 +0100
+++ ./source3/smbadduser.in	2006-11-01 15:39:54.000000000 +0100
@@ -61,7 +61,7 @@
 	endif
 
 	echo "Adding: $unix to $smbpasswd"
-	/usr/bin/smbpasswd -a -n $unix
+	$prefix/bin/smbpasswd -a -n $unix
 	if ($unix != $ntid) then
 		echo "Adding: {$unix = $ntid} to $user_map"
 		echo "$unix = $ntid" >> $user_map
